### PR TITLE
Update pattern matches 2021-09-06

### PR DESCRIPTION
**Pull request by matentzn**

### CODE REVIEW

File: src/patterns/pattern-simple.owl [hunk 1]
```diff
@@ -59,24 +59,34 @@
...
 
     <!-- http://purl.obolibrary.org/obo/mondo/patterns/acute.yaml -->
 
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/mondo/patterns/acute.yaml">
-        <obo:IAO_0000115 rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Acute form of owl_thing.</obo:IAO_0000115>
+        <obo:IAO_0000115 rdf:datatype="http://www.w3.org/2001/XMLSchema#string">Acute form of &apos;disease&apos;.</obo:IAO_0000115>
         <terms:title rdf:datatype="http://www.w3.org/2001/XMLSchema#string">acute</terms:title>
-        <oboInOwl:hasExactSynonym rdf:datatype="http://www.w3.org/2001/XMLSchema#string">owl_thing, acute</oboInOwl:hasExactSynonym>
-        <rdfs:label rdf:datatype="http://www.w3.org/2001/XMLSchema#string">acute owl_thing</rdfs:label>
+        <oboInOwl:hasExactSynonym rdf:datatype="http://www.w3.org/2001/XMLSchema#string">&apos;disease&apos;, acute</oboInOwl:hasExactSynonym>
```
Comment (nicolevasilevsky) [inline src/patterns/pattern-simple.owl:88]: is this &apos; okay?

Comment (matentzn) [reply src/patterns/pattern-simple.owl:88]: In pattern-simple.owl yes - not in tags.

**Comment by matentzn** [comment]
These cases are all in the pattern ontology files which we don't really need.

**Comment by nicolevasilevsky** [comment]
ok! Should I merge this then?

**Comment by matentzn** [comment]
Sure!

**Review by nicolevasilevsky — APPROVED** [review]

**Pull request merged**